### DISCUSSION TITLE
Weird is app icon missing

### DISCUSSION THREAD

**Discussion by William150722** [LawnchairLauncher/lawnchair#3059] [General]
![Screenshot_20221027-101801_One UI Home](https://user-images.githubusercontent.com/111726325/198175807-67ff561c-fcb8-4465-adbd-1bfb9e7e80fe.jpg)

I downloaded the Lawnchair in app store. once i set Lawnchair as default launcher.

![Screenshot_20221027-101811_Lawnchair](https://user-images.githubusercontent.com/111726325/198175827-239b9295-d2c7-421d-b08c-f1da8fd62d62.jpg)

Notice that the Lawnchair app icon is missing, is it a bug? Only left is Lawnchair Setting App Icon

I checked the code there is no app icon hiding logic inside.

**Comment by SuperDragonXD** [comment]
This is intentional behavior. It's kinda useless to have an icon that goes back to the home screen.

**Reply by William150722** [reply]
@SuperDragonXD noted with thanks.

**Comment by William150722** [comment]
Hi, can help to delete this discussion. Thanks.
